packages/html-reporter/src/index.tsx

```diff
@@ -17,7 +17,7 @@
 import type { HTMLReport } from './types';
 import type zip from '@zip.js/zip.js';
 // @ts-ignore
-import zipImport from '@zip.js/zip.js/dist/zip-no-worker-inflate.min.js';
+import * as zipImport from '@zip.js/zip.js/lib/zip-no-worker-inflate.js';
```

Comment:
According to https://github.com/gildas-lormeau/zip.js/tree/master/dist:

> Warning: These files are not compatible with ES modules, i.e. they cannot be imported with import. Instead, import index.js in the root folder of the project or one of the files prefixed with zip- in the [/lib](https://github.com/gildas-lormeau/zip.js/blob/master/lib) folder (e.g. /lib/zip-no-worker-inflate.js).

Reply:
Could you share the report sizes before and after this change?

Reply:
![](https://user-images.githubusercontent.com/1091472/211441577-c715a56a-ab6e-4d89-aec0-5f8767452f37.png)